pkg/config/templates/cluster.go

```diff
@@ -53,6 +53,8 @@ spec:
     version: {{ .Options.KubeVersion }}
     clusterName: cluster.local
   network:
+    # multus support. https://github.com/k8snetworkplumbingwg/multus-cni
+    # for example: plugin: calico-multus , plugin: flannel-multus , plugin: cilium-multus
```

Comment:
I think it is better to use a parameter alone to enable `multus-cni`. such as: `enableMultusCNI: true`
If the configuration is mixed with other network plugins, the subsequent addition of new network plugins will also involve `multus-cni`.

Reply:
You are right. I can't agree with you more

Reply:
Does the `kk`  cli  need to add this parameter?

Reply:
I think the current configuration file supports it. After all, cli does not currently support specifying calico, flannel, etc.

Reply:
ok